templates/footer-wordads.php

```diff
@@ -0,0 +1,8 @@
+<div class="adspace">
```

Comment:
Yeah, this is just a place holder, so I’ll remove it.

How will ads be supported going forward?